src/components/Jobs/JobDetails.vue

```diff
@@ -0,0 +1,119 @@
+<template>
+  <div>
+    <template v-if="job">
+      <v-container>
+        <v-layout column>
```

Comment:
The job-details header doesn't seem to break properly on mobile screens.

![image](https://user-images.githubusercontent.com/64745/57225991-2985d300-700e-11e9-8f73-c3def44f4eaa.png)